cli/src/pcluster/config/cluster_config.py

```diff
@@ -1450,6 +1474,9 @@ def _register_storage_validators(self):  # noqa: C901 FIXME: function too comple
                     existing_storage_count["fsx"] += 1
                     existing_fsx.add(storage.file_system_id)
                     self._register_validator(FeatureRegionValidator, feature=Feature.FSX_ONTAP, region=self.region)
```

Comment:
According to the feature design we decided not to add any feature/region validation, so you're doing right.
Nevertheless, I'm seeing now that we are actually doing that for other storage types, so I suggest to have a consistent approac. If you agree with that, let's add the feature/region validator in a separate PR.

In case we dice to remove such validation, we should do that consistently for every storage type, but let's keeo the things aligned.

Reply:
Sure